test/acceptance/UserManagement/LoginService/requirements.txt

```diff
@@ -0,0 +1,8 @@
+eoepca-scim==2.5.0
```

Comment:
We need to collect these Configuration Item lists in one file per BB and one central file per system. See https://eoepca.atlassian.net/browse/EOEPCA-113